src/i18n/index.js

```diff
@@ -57,7 +57,10 @@ class I18n {
         break
 
       default:
-        throw new Error(`I18n._loadResources(): unknown language: ${lang}`)
+        throw new Error(globalize('i18n.error', {
```

Comment:
leave the previous version. This place does not localize

Reply:
done